### DISCUSSION TITLE
Details on feeding data to /Set/EVMeter

### DISCUSSION THREAD

**Discussion by stomko11** [dingo35/SmartEVSE-3.5#156] [Q&A]
Hello. I have been looking around, and I configured my mains meter with proper values via MQTT. However I am struggling to find info on how to feed data for /Set/EVMeter

![image](https://github.com/user-attachments/assets/e144df53-3acf-4c58-8afb-eecf6317fce8)
![image](https://github.com/user-attachments/assets/743c9adc-fab2-4db6-a036-c1bc21016d8b)

I found some info here for current, but it does not mention anything about energy: https://github.com/dingo35/SmartEVSE-3.5/blob/master/docs/REST_API.md

Thank you

**Comment by dingo35** [comment]
From the manual:

"You can find test scripts in the test directory that feed EV and MainsMeter data to your MQTT server."

Off of the top of my head, where /Set/MainsMeter needs L1:L2:L3 as arguments, /Set/EVMeter needs L1:L2:L3:Power:Energy as arguments. But check the test scripts...
